src/ModuleList/Quiz/Quiz.tsx

```diff
@@ -1,22 +1,78 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useParams } from 'react-router-dom';
+import { SimpleGrid, Box, Flex } from '@chakra-ui/core';
+import { SubmitQuiz, CheckIcon, IncorrectIcon } from './styled';
+import { useQuizMeta } from 'src/common/api/curriculum/quiz';
+import { Question } from 'src/common/api/curriculum/quiz/requests';
 
 interface Params {
-  moduleId: string;
   quizId: string;
 }
 
+interface StringMap {
+  [key: string]: boolean;
+}
+
 const Quiz = () => {
-  const { moduleId, quizId } = useParams<Params>();
+  const { quizId } = useParams<Params>();
+
+  const { data } = useQuizMeta(Number(quizId));
+
+  const [questionIndexesAnsweredCorrectly, setQuestionIndexesAnsweredCorrectly] = useState<StringMap>({});
+  const [quizIsSubmitted, setQuizIsSubmitted] = useState<boolean>(false);
+
+  const handleAnsweredQuestion = (q: Question, qIndex: number, optionIndex: number) => {
+    return (e: React.ChangeEvent<any>) => {
```

Comment:
Since you don't use `e` you don't actually need to accept it as an argument. 

```suggestion
    return () => {
```